packages/components/src/theme/colorUtils.test.ts

```diff
@@ -0,0 +1,14 @@
+import { colorValueStyle } from './colorUtils';
+
+describe('ColorValues', () => {
+  test('should return the correct color style', () => {
+    // dh-color variables
+    expect(colorValueStyle('blue-1000')).toBe('var(--dh-color-blue-1000)');
```

Comment:
This is a good case for `it.each` since you have an array of all possible values, easy to incorporate

Reply:
Meh, this is less code. If I was doing more in each check or setup sure.